mqtts/Dockerfile

```diff
@@ -18,5 +18,4 @@ COPY mosquitto.conf /etc/mosquitto/mosquitto.conf
 # Start the MQTTS daemon during container startup
 RUN mkdir /etc/service/mosquitto
 COPY mosquitto.sh /etc/service/mosquitto/run
-RUN chmod +x /etc/service/mosquitto/run
-
+RUN chmod +x /etc/service/mosquitto/run
```

Comment:
Ok to delete line 22, but line 21 *must* end in a newline.

Reply:
It's done.